plugin-api/src/main/java/org/sonar/api/issue/Issue.java

```diff
@@ -255,4 +255,9 @@ public interface Issue extends Serializable {
    * @since 5.1
    */
   Collection<String> tags();
+
+  /**
+   * @since 9.17
+   */
```

Comment:
Maybe worth clarifying what happens when no code variants are set? I assume the list will be empty but not null?